highfidelity/hifi
# Added method to TestScriptingInterface to clear caches

# Description
A single scripting command has been added to the Test Scripting Interface.  This command is equivalent to the Interface menu command Reload Content.

The command is useful for testing, as it provides more consistent results.

# Testing
1. Run Interface with a long-running test script.
2. Open the Scripting Console
3. After the domain has loaded, run Test.clearCaches(); from the console
4. Verify that the content is cleared and then reloads.
